octopusdeploy_framework/resource_project_expand.go

```diff
@@ -138,7 +139,7 @@ func expandProject(ctx context.Context, model projectResourceModel) *projects.Pr
 }
 
 func expandGitLibraryPersistenceSettings(ctx context.Context, model gitLibraryPersistenceSettingsModel) projects.GitPersistenceSettings {
-	url, _ := url.Parse(model.URL.ValueString())
+	gitUrl, _ := url.Parse(model.URL.ValueString())
```

Comment:
code analysis was complaining about the name being the same as the imported package